### DISCUSSION TITLE
data key not present in the perfusion_inference_sd_v2.yaml.

### DISCUSSION THREAD

**Discussion by shrutidhange** [ChenDarYen/Key-Locked-Rank-One-Editing-for-Text-to-Image-Personalization#15] [Q&A]
Hi Team,  
        Great work! I am trying your code. I was able to run your code and try on some customization. v 1 but with v2 I am facing issues.
In the configs/perfusion_inference_sd_v2.yaml I don't see the property to mention the data key which is present in the configs/perfusion_custom.yaml.  
       I tried to add this below data key in the yaml but it throws error. Please let me know why data key is not present in the perfusion_inference_sd_v2.yaml. Please help me on this.

`data:
  target: main.DataModuleFromConfig
  params:
    batch_size: 4
    num_workers: 10
    wrap: true
    train:
      target: perfusion.dataset.PersonalizedBase
      params:
        data_root: '/path/to/your/images/'
        flip_p: 0.5  # 0. when asymmetric
        prompt_template:

lightning:
  callbacks:
    image_logger:
      target: main.ImageLogger
      params:
        frequency_base: 5
        batch_frequency: 25
  modelcheckpoint_step:
    params:
      every_n_train_steps: 50
  trainer:
    max_steps: 400
    log_every_n_steps: 1
    accumulate_grad_batches: 4
    precision: 16`

**Comment by AkshayJainX** [comment]
@ChenDarYen, I am facing the same issue. I'll be eternally grateful if I can implement the v2 model. 
Thanks @ChenDarYen and @shrutidhange in advance.
